docs/overview/overview.md

```diff
@@ -1,45 +1,31 @@
 ---
 title: 'Office Scripts in Excel on the web'
-description: 'The prerequisites and environment information for Office Scripts for Excel on the web.'
-ms.date: 12/05/2019
+description: 'A brief introduction to the Action Recorder and Code Editor for Office Scripts.'
+ms.date: 12/12/2019
 localization_priority: Normal
 ---
 
-# Office Scripts in Excel on the web
+# Office Scripts in Excel on the web (preview)
 
 [!INCLUDE [Preview note](../includes/preview-note.md)]
 
-Office Scripts in Excel on the web let you automate tasks. You can record your Excel actions with the [Action Recorder](https://aka.ms/makersdogfood), which creates a script. You can also create and edit scripts with the Code Editor. This series of documents teaches you to use the Code Editor to make or update your own scripts by learning the fundamentals of Office Scripts.
+Office Scripts in Excel on the web let you automate tasks. You can record your Excel actions with the Action Recorder, which creates a script. You can also create and edit scripts with the Code Editor. This series of documents teaches you how to use these tools. You'll be introduced to the Action Recorder and see how to record your frequent Excel actions. You'll also learn how to make or update your own scripts with the Code Editor.
 
-> [!NOTE]
-> If you're looking to develop an Office Add-In for Excel, please visit our [quick start](/office/dev/add-ins/quickstarts/excel-quickstart-jquery) or [tutorial](/office/dev/add-ins/tutorials/excel-tutorial). Learn more about the differences between Office Scripts and Office Add-ins in [the Differences between Office Scripts and Office Add-ins article](../resources/differences-scripts-add-ins.md).
+<img alt="A gif showing a script being recorded and replayed on different worksheets." src="../images/overview.gif" />
```

Comment:
You should be able to use a .gif in the docs (see the [Outlook mobile add-ins page](https://docs.microsoft.com/en-us/outlook/add-ins/outlook-mobile-addins#what-makes-a-good-scenario-for-mobile-add-ins)) with normal Markdown notation.